terraform/modules/jenkins/ec2.tf

```diff
@@ -0,0 +1,41 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_instance" "jenkins2" {
```

Comment:
Could change the naming of things to just 'jenkins'?

Reply:
I think each time we need a new box we'll need a similar process of running two in parallel, unless it's an emergency. So I think it's likely we'll just keep incrementing the number e.g. jenkins3 etc